spec/request/appeals_request_spec.rb

```diff
@@ -78,8 +72,8 @@
     context 'with a server error' do
       it 'returns a 502 and logs an error level message' do
         VCR.use_cassette('appeals/server_error') do
-          expect_any_instance_of(Appeals::Service).to receive(:log_message_to_sentry)
-            .with('Appeals::Service handled an unexpected 500', :error, anything)
+          # expect_any_instance_of(Appeals::Service).to receive(:log_message_to_sentry)
+          #   .with('Appeals::Service handled an unexpected 500', :error, anything)
```

Comment:
why is this one commented out instead of deleted?

Reply:
👍good catch, forgot I commented that one out